knative-operator/deploy/resources/dashboards/serving/grafana-dash-knative-serving-service-queue-proxy.yaml

```diff
@@ -0,0 +1,638 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: grafana-dash-knative-serving-service-queue-proxy
+  namespace: openshift-config-managed
+  labels:
+    console.openshift.io/dashboard: "true"
+data:
+  eventing-source-dashboard.json: |+
```

Comment:
`eventing-source-dashboard.json` should be changed?

Reply:
Something went wrong when pushing the file renaming thing. I changed it.